### PR TITLE
added typography component

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: css/common.css [hunk 1]
```diff
@@ -128,14 +128,13 @@ h1{
     font-size:4.5rem;
     margin-left:0;
 }
-h2{
+.nav-sub-header{
     font-size: 1.8rem;
     color: #6c757d;
```
Comment (itsmebipul00) [inline css/common.css]: you could use var

**Review by itsmebipul00 — APPROVED** [review]

**Review by Muzammil-khan710 — APPROVED** [review]
Use var in colors.. else is good to go !

**Review by DivyanshSareen — APPROVED** [review]
Good to merge. Noticed `margin: 0` and `padding: 0` at multiple places, would suggest removing the redundant ones.

File: components/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.typography-headings{
+    text-align: left;
+    margin-left: 0;
```
Comment (devAmlan) [inline components/typography/typography.css]: No need to use this line I think, bcz u set margin:0

File: components/typography/typography.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    font-size: 2.3rem;
+}
+.para-xsmall{
+    font-size: 1.5rem;
+}
+.typography-styles{
+    font-size: 2.3rem;
+    margin: 0;
+}
+
```
Comment (zoef-yt) [inline components/typography/typography.css]: Remove spacing

**Review by zoef-yt — APPROVED** [review]
Good to merge with the changes.

**Comment by zoyatanweer** [comment]
Thankyou for all the reviews. Made the changes.

**Pull request merged**